extensions/basic_auth/config/gateway-filter.yaml

```diff
@@ -12,12 +12,30 @@ spec:
         filterChain:
           filter:
             name: envoy.http_connection_manager
-      proxy:
```

Comment:
should we keep the proxy version match for 1.9?

Reply:
Done.